prometheus/base/prometheus.rules.yaml

```diff
@@ -1,13 +1,12 @@
-apiVersion: monitoring.coreos.com/v1
-kind: PrometheusRule
-metadata:
-  labels:
-    openshift.io/prometheus-rule-evaluation-scope: leaf-prometheus
-  name: o11y-example-rule
-  namespace: o11y
-spec:
-  groups:
-    - name: example
-      rules:
-        - record: o11y_prometheus_http_requests_total
-          expr: sum by (code) (prometheus_http_requests_total)
+
+groups:
```

Comment:
Why are we removing k8s settings? I thought @yftacherzog #9 fix that issue for now?
do note we have a patch on Infra Deployments that pull updates for the rules file

Reply:
Its like that because #9 is not merged yet, and I wanted you to take a look at the work in progress